.env

```diff
@@ -13,4 +13,4 @@ DB_NAME=nijobs
 SESSION_SECRET=O Rui foi membro do IEEE.
 
 # Specifies the port in which the app will be exposed
-PORT=8000
+PORT=8087
```

Comment:
Why this change? Anything in particular?

Reply:
Collided with a lot of stuff that opens listens on 8000 by default.

Reply:
:+1: